prism.js

```diff
@@ -0,0 +1,77 @@
+const util = require('util');
+
+const refractor = require('refractor/core.js');
+const languages = require('prism-languages');
+
+var remark = require('remark');
+var parse = require('remark-parse');
+var htmlRemark = require('remark-html');
+
+const visit = require('unist-util-visit');
+var inspect = require('unist-util-inspect');
+var find = require('unist-util-find');
+var findAllBetween = require('unist-util-find-all-between');
+
+refractor.register(require('./jsLinks.js'));
+refractor.register(require('refractor/lang/json'))
+
+var fs = require('fs');
+
+var file = fs.readFileSync(__dirname + '/src/content/configuration/' + 'index.mdx', { encoding: 'utf8' });
+
+remark()
+  .use(parse)
+  .use(attacher)
+  .use(htmlRemark)
+  .process(file, function(err, file) {
+    if (err) throw err;
+    if (file) {
+      fs.writeFileSync(
+        './index.html',
+        `<link rel="stylesheet" href="https://webpack.js.org/142727ee4bde248bb419.css" />
+      <div class="page__content">` +
+          file.contents +
+          `</div>`
+      );
+    }
+  });
+
+function attacher({ include, exclude } = {}) {
+  return transformer;
+
+  function transformer(tree, file) {
+    visit(tree, 'code', visitor);
+
+    function visitor(node) {
+      const { lang } = node;
+
+      if (!lang || (include && !~include.indexOf(lang)) || (exclude && ~exclude.indexOf(lang))) {
```

Comment:
cant we just use `.includes` instead of `.indexOf` trickery?

Reply:
Yep